app/CocoaPods/CPUserProject.m

```diff
@@ -266,6 +271,13 @@ - (void)appendTaskOutput:(NSString *)rawOutput;
   } else {
     self.taskOutput = attributedOutput;
   }
+
+  // Keep the text view at the bottom if it was previously, otherwise restore the previous position.
+  if (scrolledToBottom) {
+    [self.progressOutputView scrollToEndOfDocument:self];
```

Comment:
No, it looks like the behavior Terminal has when printing stuff out

> On Nov 4, 2015, at 19:15, Kyle Fuller <notifications@github.com> wrote:
> 
> In app/CocoaPods/CPUserProject.m:
> 
> > @@ -266,6 +271,13 @@ - (void)appendTaskOutput:(NSString *)rawOutput;
> >    } else {
> >      self.taskOutput = attributedOutput;
> >    }
> > +
> > +  // Keep the text view at the bottom if it was previously, otherwise restore the previous position.
> > +  if (scrolledToBottom) {
> > +    [self.progressOutputView scrollToEndOfDocument:self];
> Is this animated? The documentation doesn't really mention it either way.
> 
> —
> Reply to this email directly or view it on GitHub.
>